File: src/main/java/com/github/njuro/jard/config/security/LogoutSuccessHandler.java
package com.github.njuro.jard.config.security;

import org.springframework.security.web.authentication.logout.SimpleUrlLogoutSuccessHandler;
import org.springframework.stereotype.Component;

@Component
public class LogoutSuccessHandler extends SimpleUrlLogoutSuccessHandler {

  public LogoutSuccessHandler() {
    setRedirectStrategy(new NoRedirectStrategy());
  }
}

File: src/main/kotlin/com/github/njuro/jard/rewrite/config/security/LogoutSuccessHandler.kt
package com.github.njuro.jard.rewrite.config.security

import com.github.njuro.jard.config.security.NoRedirectStrategy
import org.springframework.security.web.authentication.logout.SimpleUrlLogoutSuccessHandler
import org.springframework.stereotype.Component

@Component
class LogoutSuccessHandler : SimpleUrlLogoutSuccessHandler() {
    init {
        redirectStrategy = NoRedirectStrategy()
    }
}